NFT-Collection.md

```diff
@@ -792,7 +792,7 @@ export default function Home() {
         <div>
           <h1 className={styles.title}>Welcome to Crypto Devs!</h1>
           <div className={styles.description}>
-            Its an NFT collection for developers in Crypto.
+            It's an NFT collection for developers in Crypto.
```

Comment:
Since this is inside a rendering tag, you should do `It&#39;s`. Please test this out first.